src/tabpfn/model_loading.py

```diff
@@ -569,6 +573,30 @@ def _resolve_model_version(model_path: ModelPath | None) -> ModelVersion:
     return ModelVersion.V2
 
 
+def _log_model_config(
+    model_paths: list[Path],
+    which: Literal["classifier", "regressor"],
+    version: ModelVersion,
+) -> None:
+    """Set the model config (model_path and model_version) for anonymous
+    usage telemetry.
+
+    Args:
+        model_paths: The path(s) to the model.
+        which: The type of model ('classifier' or 'regressor').
+        version: The model version (currently only 'v2' or 'v2.5').
+    """
+    if len(model_paths) != 1:
+        return
+
+    model_type = ModelType(which)
+    model_source = _get_model_source(version, model_type)
+
+    path: Path = model_paths[0]
+    if path.name in model_source.filenames:
+        set_model_config(path.name, version.value)
```

Comment:
Nice, this looks safe! Maybe an extra bucket for everything else?

```suggestion
        set_model_config(path.name, version.value)
    else:
        set_model_config("OTHER", version.value)
```